src/leo-server-app/src/routes/satellite.ts

```diff
@@ -1,16 +1,50 @@
+import * as dotenv from "dotenv";
+import { TLEResponse } from "../types/satellites";
+
+dotenv.config({ path: `.env.local`, override: true });
+
 const express = require("express");
+var spacetrack = require("spacetrack");
+
 const Satellite = require("../models/satellite");
 const User = require("../models/user");
 const satellite = require("satellite.js");
 
 const router = express.Router();
 router.use(express.json());
 
-// NEUDOSE TLE from satnogs
-var tleLine1 =
-    "1 56315U 98067VG  23289.56381294  .00217706  00000+0  12767-2 0  9999",
-  tleLine2 =
-    "2 56315  51.6294  74.1402 0004981 252.1253 107.9204 15.77094333 27340";
+spacetrack.login({
+  username: process.env.SPACE_TRACK_USERNAME,
+  password: process.env.SPACE_TRACK_PASSWORD,
+});
+
+let tleLine1: string;
+let tleLine2: string;
+
+// BDSAT-2 TLE from Space-Track accessed 12/25/2023
+var defaultTleLine1 =
+    "1 55098U 23001CT  23359.66872105  .00021921  00000-0  89042-3 0  9991",
+  defaultTleLine2 =
+    "2 55098  97.4576  58.0973 0014812  57.5063 302.7604 15.24489013 54199";
+
+spacetrack
```

Comment:
Does this endpoint get called periodically or only when the server side first boots up? It seems like the TLE data is captured once from spacetrack which is then used in some of the other endpoints. I was wondering does that info need to be refetched anytime we call '/getSatelliteInfo' or '/getNextPasses'?

Reply:
updates when server side boots up, need to actually host everything on a server to test if it periodically refreshes (should pull every day). I'd avoid refetching the info on every api call as per space-track's api use guidelines: https://www.space-track.org/documentation#/api

however we are interfacing with it through a library so it could get around it? i would rather avoid a situation where this happens

Reply:
Ah I see, LGTM then, I guess would there be a situation where we would need to periodically refetch it? If not, then I guess we don't really need to worry about it

Also, I was reading through the api doc and it says recommended frequency for fetching TLE data is 1/hour. I was thinking maybe we could make a future pr to call the endpoint every 1 or 2 hours, wanted to get your thoughts?

Reply:
we do need to periodically refresh it (#109), but we first need to host it on the server (#107) for some kind of permanence